packages/remark-mdx/readme.md

```diff
@@ -22,6 +22,16 @@ It’s used in [MDXjs][].
 npm install remark-mdx
 ```
 
+### Installing Other Plugins
+
+You'll probably need other plugins such as [**remark-parse**][remark-parse].
+Make sure you are using compatible versions of such plugins.
+In case of [**remark-parse**][remark-parse] currently version `^8.0.3` is supported (the latest is `9.0.0`).
```

Comment:
`main` branch is documenting mdx 2 which is compatible with remark 9 plugins.

Reply:
this is true. I just didn't find any other place specifically talking about **remark-mdx** package (and not the whole **mdx** project) besides this readme file.